examples/Box2D.Window/SimulationWindow.cs

```diff
@@ -245,7 +245,7 @@ public void DrawCircle(Vec2 center, float radius, Color color)
             });
         }
 
-        public void DrawSolidCircle(Vec2 center, float radius, Vec2 axis, Color color)
+        public void DrawSolidCircle(System.Numerics.Vector2 center, float radius, System.Numerics.Vector2 axis, Color color)
```

Comment:
I'd just just `Vector2` instead of longer `System.Numerics.Vector2` because it helps to keep it shorter line here. It's up to you to decide! :)